dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1220,3 +1220,23 @@ CWWKD1120.cursor.keyword.mismatch.explanation=Keywords such as GROUP BY, HAVING,
  pagination.
 CWWKD1120.cursor.keyword.mismatch.useraction=Rewrite the query to include only \
  the clauses: SELECT, FROM, and WHERE.
+
+CWWKD1121.record.lacks.id=CWWKD1121E: The {0} record cannot be used as an entity \
+ because it lacks a unqiue identifier. One of the record components ({1}) must \
+ be named id, have a name that ends with _id, Id or ID, or be of the type \
+ java.util.UUID.
+CWWKD1121.record.lacks.id.explanation=Entities are required to have a unique \
+ identifier attribute.
+CWWKD1121.record.lacks.id.useraction=Add or rename a record component to \
+ follow one of the stated conventions for assigning a unique identifier.
+
+CWWKD1122.entity.lacks.id=CWWKD1122E: The {0} class cannot be used as an \
+ entity because it lacks a unique identifier. To assign a unique identifier, \
+ ensure the {1} feature is enabled and the entity class is annotated \
+ @jakarta.persistence.Entity and has a field or getter method annoated \
+ @jakarta.persistence.Id. Alternatively, use a META-INF/orm.xml file to \
+ define the entity and its unique identifier.
```

Comment:
```suggestion
CWWKD1122.entity.lacks.id=CWWKD1122E: The {0} class cannot be used as an \
 entity because it lacks a unique identifier. One of the available attributes ({1}) \
 on the class must be named id, have a name that ends with _id, Id or ID, or be of the type \
 java.util.UUID.
```

Since this error message will only appear for unannotated entities.